client/README.md

```diff
@@ -0,0 +1,77 @@
+# SDK Client
+
+## Instruction
+
+### Remove remaining data(if exists)
+
+```bash
+> rm -r ~/.gaiad
+> rm -r ~/.gaiacli
+```
+
+### Installation
+
+```bash
+> go get -u github.com/cosmos/cosmos-sdk
+package github.com/cosmos/cosmos-sdk: no Go files in /home/mossid/go/src/github.com/cosmos/cosmos-sdk
```

Comment:
?

Reply:
Is this line an error you encountered? Why is it on the tuto?

Reply:
Is this line an error you encountered? Why is it on the tuto?

Reply:
maybe link to the quick install script? https://github.com/cosmos/cosmos-sdk/tree/rigel/spec-staking/scripts